### PR TITLE
paloalto + ibm updates

### PR DESCRIPTION
**Pull request by zer010bs**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -57,6 +57,19 @@ Log4j impact manufacturers and components summary from the Internet community. W
...
+#### IBM Qradar SIEM
+
+- /opt/qradar/support/mod_log4j.pl
+
+- [logging](https://www.ibm.com/mysupport/s/question/0D50z00006PEIeQCAX/qradar-qradarlog-and-qradarerror-slf4j-this-version-of-slf4j-requires-log4j-version-1212-or-later?language=en_US)
+
+
+#### PaloAlto Panorama
+
+- [logging](https://docs.paloaltonetworks.com/oss-listings/panorama-oss-listings/panorama-10-0-open-source-software-oss-listing.html)
```
Comment (Firminator) [inline README.md:71]: And v10.1 as well: https://docs.paloaltonetworks.com/oss-listings/panorama-oss-listings/panorama-10-1-open-source-software-oss-listing.html

Or more genrically https://docs.paloaltonetworks.com/search.html#q=log4j

**Comment by YfryTchsGD** [comment]
Nice job. The current update method will cause conflicts, so I updated it manually. Thank you mr.

**Comment by zer010bs** [comment]
cool, just re-forked (had conflicts as well) and will keep you updated if i see new things popping up

**Comment by zer010bs** [comment]
wil go and workj with the issues/ticketsystem. found a couple more protentailly affected
^^cough^^ORACLE^^ĉough

**Pull request closed without merging**